pkg/reconciler/v1alpha1/clusteringress/clusteringress.go

```diff
@@ -69,10 +85,13 @@ func NewController(
 	impl := controller.NewImpl(c, c.Logger, "ClusterIngresses", reconciler.MustNewStatsReporter("ClusterIngress", c.Logger))
 
 	c.Logger.Info("Setting up event handlers")
-	clusterIngressInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
-		AddFunc:    impl.Enqueue,
-		UpdateFunc: controller.PassNew(impl.Enqueue),
-		DeleteFunc: impl.Enqueue,
+	clusterIngressInformer.Informer().AddEventHandler(cache.FilteringResourceEventHandler{
+		FilterFunc: filterAnnotation(networking.IngressClassAnnotationKey, networking.IstioIngressClassName, true),
```

Comment:
I think the filter func should also be added into the virtualService informer loop below